lib/init.coffee

```diff
@@ -252,14 +252,17 @@ module.exports =
           messages = @parseSchemaMessages(textEditor, output)
         if messages.length
           for message in messages
-            message.type = 'Error'
-            message.text = message.text + ' (' + schemaUrl + ')'
-            message.filePath = textEditor.getPath()
+            message.severity = 'error'
+            message.excerpt = message.excerpt + ' (' + schemaUrl + ')'
```

Comment:
`schemaUrl` might contain a relative path and not necessarily an absolute URL. I am not sure if `message.url` works with that? If not I would rather keep it as is for now and keep this improvement for another PR.